sopel/modules/reddit.py

```diff
@@ -427,8 +435,9 @@ def get_channel_spoiler_free(bot, trigger):
     if not channel:
         channel = trigger.sender
         if channel.is_nick():
-            return bot.say('{}getspoilfree with no channel param is only permitted '
-                           'in channels'.format(bot.config.core.help_prefix))
+            bot.reply('{}getspoilfree with no channel param is only permitted '
+                      'in channels'.format(bot.config.core.help_prefix))
```

Comment:
```suggestion
                      'in channels.'.format(bot.config.core.help_prefix))
```

Something something proper grammar something. (Yeah, I'm saying let's fix these little things "as long as the line's already getting touched".)

Reply:
It's the right moment to do it, too.